psi4/src/read_options.cc

```diff
@@ -1403,8 +1403,6 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_int("PK_MAX_BUCKETS", 500);
         /*- Select the PK algorithm to use. For debug purposes, selection will be automated later. !expert -*/
         options.add_str("PK_ALGO", "REORDER", "REORDER YOSHIMINE");
```

Comment:
After this PR, does this keyword name make sense?

Reply:
I think so -- it's still the only way to choose btwn reorder and yoshimine when scf_type=PK. Am I missing your point?

Reply:
If PK is the algorithm and the subalgorithm is in core or out-of-core, is this a sub-sub-algorithm?

Reply:
> EDIT: docstring might be edited to mention if you want PK+INCORE, use subtype.

This is a good idea! I'd be happy to put that in.

Reply:
> If PK is the algorithm and the subalgorithm is in core or out-of-core, is this a sub-sub-algorithm?

One could frame it as such, yes - `YOSHIMINE` and `REORDER` are effectively sub-sub-algorithms to the out-of-core PK subalgorithm.

Reply:
> is this a sub-sub-algorithm?

Yeah, that's what I didn't like about it. But the most logical fix I could devise (see above) is expanding scf_type, so this is the least intrusive for now.

Reply:
What about collapsing `PK_ALGO` into `SCF_SUBTYPE`? So for PK, allowed options are `INCORE`, `OOC_REORDER`, and `OOC_YOSHIMINE`?

Reply:
If there's enough support for folding `PK_ALGO` into `SCF_SUBTYPE`, however, I wouldn't mind doing that for this PR. I would also be happy to tackle Lori's proposal, but I'd rather that be for another PR.

Reply:
I'm still for folding `PK_ALGO` into `SCF_SUBTYPE`, thanks for considering.

Agreed that Lori's proposal is beyond-scope.